src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -172,14 +170,16 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * Submits manual textAssessments for a given result and notify the user if it's before the Assessment Due Date
+     *
+     * POST exercise/:exerciseId/result/:resultId/submit : Submits manual textAssessments for a given result
+     *     and notify the user if it's before the Assessment Due Date
      *
      * @param exerciseId the exerciseId of the exercise which will be saved
      * @param resultId the resultId the assessment belongs to
      * @param textAssessment the assessments which should be submitted
      * @return 200 Ok if successful with the corresponding result as a body, but sensitive information are filtered out
      */
-    @PutMapping("/exercise/{exerciseId}/result/{resultId}/submit")
+    @PostMapping("/text-submissions/exercise/{exerciseId}/result/{resultId}/submit")
```

Comment:
same here as above. Prepending text-submissions does not make sense.
Also this URL uses a "/" in the beginning, while other don't --> this should be consistent without "/"

Also note the difference between submit and save. Such inconsistencies are really bad in my opinion! Please have them consistent